sale_manual_delivery/tests/test_manual_delivery.py

```diff
@@ -289,7 +289,7 @@ def test_03_sale_selected_lines(self):
             'Picking should be created after "manual delivery" wizard call',
         )
         self.assertEqual(
-            len(order3.picking_ids.move_lines),
+            len(order3.picking_ids.move_line_ids),
```

Comment:
:warning: `move_lines` refers to `stock.move` whereas `move_line_ids` refers to `stock.move.line`.
I guess, here you should use `move_ids`: https://github.com/odoo/odoo/blob/16.0/addons/stock/models/stock_picking.py#L346